src/datasources/earth.js

```diff
@@ -1,18 +1,25 @@
 const { RESTDataSource } = require("apollo-datasource-rest");
+const Materials = require("../../database/models/materials-model");
 
 class EarthAPI extends RESTDataSource {
   constructor() {
     super();
     this.baseURL = `http://api.earth911.com`;
     this.apiKey = `?api_key=${process.env.earth911_secret}`;
   }
-  materialReducer(material) {
+
+  materialReducer(binInfo, material) {
     return {
       description: material.description,
       material_id: material.material_id,
-      long_description: material.long_description
+      long_description: material.long_description,
+      // "!!+" converts 0/1 into boolean
+      bin_trash: binInfo && !!+binInfo.bin_trash,
+      bin_recycle: binInfo && !!+binInfo.bin_recycle,
+      bin_compost: binInfo && !!+binInfo.bin_compost
```

Comment:
It took me a second to realize that you must have made a binInfo object containing those three properties.

Reply:
Yeah, that's actually the entire row returned from the materials table but it only uses those three entries from the object.